doc/en/user/source/data/webadmin/layers.rst

```diff
@@ -195,30 +195,71 @@ Vector layers have a list of the :guilabel:`Feature Type Details`. These include
 
-The :guilabel:`Customize attributes` checkbox opens an attribute editor allowing customization.
+The :guilabel:`Customize attributes` checkbox activates the attributes customization view.
 
 .. figure:: img/data_layers_feature_customize.png
 
    Attribute customization
 
-It is possible to:
+This view allows to:
 
+* Add a new attribute through the add attribute dialog.
+* Reset the table to the native settings, using the :guilabel:`Reset customization` link.
 * Change the order of attributes, using either the up/down arrows, or dragging the attribute row.
 * Remove an attribute using the "remove" icon at the end of the attribute row.
-* Add a new attribute, which will be computed based on the :guilabel:`Source` CQL expression.
-* Rename an attribute.
-* Add a description of the attribute, which will be visible wherever the feature type is described.
-* Change the nillability of the attribute, for example, making the attribute mandatory even if it's
-  not in the data source, and vice-versa.
-* Change the type of the attribute using the `Type` column. The most common types are available in 
-  a drop-down on editing, but it's possible to indicate any valid Java class, as long as GeoServer
-  has a converter that goes from the value produced by the :guilabel:`Source` expression to the 
-  target type (new converters can be plugged in with some Java programming).
-* Reset the table to the native settings, using the :guilabel:`Reset customization` link.
+* Edit an attribute from the edit attribute dialog, using the "edit" icon.
 
 Some of the feature type edits might result in the layer not being editable anymore, for example,
 by removing an attribute that is marked as mandatory in the data source.
 
+From the :guilabel:`Add attribute dialog` it is possible to define a new attribute for the feature type, specifying:
+
+* The name of the new attribute.
+* The type of the new attribute using the :guilabel:`Type` field. The most common types are available in a drop-down on editing,
+  but it's possible to indicate any valid Java class, as long as GeoServer has a converter that goes from the value produced by the
+  :guilabel:`Source` expression to the target type (new converters can be plugged in with some Java programming).
+* The source for the new attribute, which will be computed based on the :guilabel:`Source` CQL expression in the respective field.
+* A description for the new attribute, which will be visible wherever the feature type is described.
+* The `nillability` of the new attribute, for example, making the attribute mandatory even if it's not in the data source, and vice-versa.
+
+.. figure:: img/data_layers_feature_customize_add_attribute.png
+
+   Add attribute dialog
```

Comment:
I *really* like this change, thank you for surfacing this FeatureType model functionality that was made so long ago for app-schema.